### PR TITLE
Bug fix: Add Language constant Where is needed

### PR DESCRIPTION
**Pull request by shitalkakade24**

### CODE REVIEW

File: src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini [hunk 1]
```diff
@@ -23,59 +23,64 @@ COM_TJDASHBOARD_ERROR_MESSAGE_NOT_AUTHORISED = "You don't have enough permission
...
-COM_TJDASHBOARD_ERROR_TEXT_FOLDER_NOT_FOUND="Folder not exists"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_WIDGET_ID="Widget Id"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_SIZE="Size"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_RENDERER_PLUGIN="Renderer Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_DATA_PLUGIN="Data Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ID="Dashboard Id"
+
+;
+COM_TJDASHBOARD_SUCCESS_TEXT="Success"
+COM_TJDASHBOARD_ERROR_TEXT_METHOD_NOT_FOUND="Method not exists"
```
Comment (coolbung) [inline src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: Where are the below 4 constants being used ? If they are being used rename them as below

- Method does not exist
- Class does not exist
- File does not exist
- Folder does not exist

Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: This is not addedd by me.Don't know where it is used

Comment (coolbung) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: Ok so find out. Delete them if they're not being used.

Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: Not used anywhere.Removed.

File: src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini [hunk 2]
```diff
@@ -23,59 +23,57 @@ COM_TJDASHBOARD_ERROR_MESSAGE_NOT_AUTHORISED = "You don't have enough permission
...
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_SIZE="Size"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_RENDERER_PLUGIN="Renderer Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_DATA_PLUGIN="Data Plugin"
+COM_TJDASHBOARD_FORM_LBL_DASHBOARD_ID="Dashboard Id"
+
+;Error For No data/Widget
+COM_TJDASHBOARD_WIDGETS_NOTSHOW_ERROR_MESSAGE="No widgets found to show"
+COM_TJDASHBOARD_WIDGETS_NO_RENDERER_DATA_ERROR_MESSAGE="No data to render"
+COM_TJDASHBOARD_WIDGETS_NO_DATA_ERROR_MESSAGE="No data"
+COM_TJDASHBOARD_NO_DATA_AVAILABLE_MESSAGE="No data Available"
```
Comment (coolbung) [inline src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: Above 3 also seem to be duplicates. Can you check which ones are needed and remove unused ones. If all 3 are used see if you can just keep 1.

Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: its not duplicates & It is used.But it is added in  Frontend Language as well as Backend language file

Comment (coolbung) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: But these have the same meaning. Can you find out where all they are used and just keep one ?

Comment (shitalkakade24) [reply src/components/com_tjdashboard/site/languages/en-GB/en-GB.com_tjdashboard.ini]: done

**Pull request merged**